lambda_functions/healthcheck/healthcheck.py

```diff
@@ -18,12 +13,17 @@ def check_web_url_health(url, expected_string, env):
     :type expected_string: String
     :param env: This is for us to distinguish the message between the dev, preprod and prod environments
     :type env: string
-    :raises logger.error: will log an error to the lambda log group
-    :raises logger.error: variable
+    :raises RuntimeError: will raise a runtime error if appropriate
+    :raises RuntimeError: error
+    :raises Exeption: will raise a runtime error if appropriate
+    :raises Exeption: error
     """
 
-    # Ping the url
-    response = requests.get(url, timeout=5)
+    try:
+        # Ping the url
+        response = requests.get(url, timeout=5)
+    except Exception as e:
```

Comment:
Can you restructure so that all the code you expect to run (i.e 28-59 lies in the try block. It is much clearer what the flow of code is that way.

Reply:
we do not want that though? The lambda must fail if there is an issue and a try/catch block would prevent this?

Reply:
Otherwise the alarm will not trigger